srcalc/src/main/java/gov/va/med/srcalc/domain/ReferenceNote.java

```diff
@@ -28,8 +28,12 @@ public String getLocalTitle()
         return fLocalTitle;
     }
 
+    /**
+     * Set the local title for this reference note. Necessary for XML binding.
+     * @param fLocalTitle the name of the local title
```

Comment:
It's OK to leave them, but in my opinion setters don't need `@param` docs unless they provide any extra meaning.